ui/components/HomeCard.tsx

```diff
@@ -10,10 +10,12 @@ export default function HomeCard({
 }: any) {
   return (
     <Link href={href} passHref>
-      <div className="card bg-base-100 shadow-md transition ease-in-out hover:-translate-y-1 cursor-pointer">
-        <div className="card-body items-stretch items-center ">
+      <div className="card bg-base-100 shadow-md transition ease-in-out hover:-translate-y-1 cursor-pointer dark:bg-slate-600">
```

Comment:
@amochuko Try reducing the brightness of the `HomeCard`, for less contrast.  For example slate-800 or something closer to the background color.

> ![](https://github.com/user-attachments/assets/45f68380-c7dd-4398-bac0-b6fa18898481)

The contrast between the card and the background should be similar as the one in light mode:

> ![](https://github.com/user-attachments/assets/35892cea-cec8-454b-8743-5a3281a839be)

Reply:
Using `slate-700` would be best to reducing the brightness of the HomeCard, for less contrast as `slate-800` is the actual background shade.

With regards to uniformity, I should also reflect that for the MainCard component?

Reply:
Yes, that looks better 👍

Reply:
> With regards to uniformity, I should also reflect that for the MainCard component?
> 

Yes, please.

Reply:
> > With regards to uniformity, I should also reflect that for the MainCard component?
> 
> Yes, please.

I've did so, and made a push earlier.